### PR TITLE
Improve translations

### PR DESCRIPTION
**Pull request by mschwemer**
Hi,

I came across some missing translations and labels.

In this PR I added them for further use  :-)

Have a good weekend.
Marcus

### CODE REVIEW

File: Resources/Private/Language/de.locallang.xlf [hunk 1]
```diff
@@ -47,27 +47,27 @@
...
 			<trans-unit id="tx_ats.registrationLinkText">
 				<source>Register here</source>
-				<target></target>
+				<target>Hier registieren</target>
 			</trans-unit>
 
 			<trans-unit id="tx_ats.returnToJob">
 				<source>Return to job</source>
-				<target></target>
+				<target>Zurück zur Stellenberschreibung</target>
```
Comment (mbrodala) [inline Resources/Private/Language/de.locallang.xlf]: There's a typo here.

File: Resources/Private/Language/de.locallang.xlf [hunk 2]
```diff
@@ -129,6 +134,10 @@
 				<source>Back</source>
 				<target>Zur&#252;ck</target>
 			</trans-unit>
+			<trans-unit id="tx_ats.button.submit">
```
Comment (mbrodala) [inline Resources/Private/Language/de.locallang.xlf]: Please add an empty line above.

File: Resources/Private/Language/locallang.xlf [hunk 3]
```diff
@@ -92,6 +92,10 @@
 			<source>Upload new file</source>
 		</trans-unit>
 
+	  <trans-unit id="tx_ats.prepend.select">
```
Comment (mbrodala) [inline Resources/Private/Language/locallang.xlf]: Please fix the indentation here.

File: Resources/Private/Templates/Application/AdditionalData/EditAdditionalData.html [hunk 4]
```diff
@@ -49,7 +49,7 @@ <h1><f:translate key="tx_ats.headline.additional_data">Additional Data</f:transl
     	</f:if>
     </f:form.validationResults>
 
-    <f:link.action action="editQualifications" controller="Application\\Qualifications" arguments="{application: application}">Back</f:link.action>
+      <f:link.action action="editQualifications" controller="Application\\Qualifications" arguments="{application: application}"><f:translate key="tx_ats.button.prev">Back</f:translate></f:link.action>
```
Comment (mbrodala) [inline Resources/Private/Templates/Application/AdditionalData/EditAdditionalData.html]: Please check the indentation here.

File: Resources/Private/Language/locallang.xlf (hunk 3 above)
Comment (mschwemer) [reply Resources/Private/Language/locallang.xlf]: Hmm .. the indentation is inconsistent most (all?) language files ... It's the same indentation as the following elements and the same indentation as the three elements above. 

IMHO it's better to fix all indentation issues at once.

Comment (mschwemer) [reply Resources/Private/Language/locallang.xlf]: I would take care ... what's your favorite indentation for xlf? There is no definition in .editorconfig ...

Comment (mbrodala) [reply Resources/Private/Language/locallang.xlf]: You are right, we'll normalize this separately.

**Comment by mbrodala** [comment]
Can you rebase your changes on top of the latest master? Thanks.

**Comment by mbrodala** [comment]
@mschwemer Can you have another look?

**Comment by pm-sschreiber** [comment]
Since we fixed a lot of translation issues since this PR, merging would be quite difficult. We will fix identation problems in a separate PR. Thanks for your work anyway.

**Pull request closed without merging**